### PR TITLE
Rename params with path_params

### LINKED ISSUE #457: Proposal to rename `params` with `path_params`
Imo, `params` don't really indicate anything meaningful. We should be referring them as `path_params` and being more explicit

### PR DESCRIPTION
**Pull request by carlosm27**
**Description**
This pull request address the proposal to rename `params` with `path_params`.

This PR fixes #457

### CODE REVIEW

**Review by sansyrox — COMMENTED** [review]
@carlosm27 , thank you for your PR but you may need to rebase it.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "robyn"
-version = "0.26.0"
+version = "0.26.000001"
```
Comment (sansyrox) [inline Cargo.toml:3]: We don't need to do this anymore. Is this instruction still present somewhere?

Comment (carlosm27) [reply Cargo.toml:3]: Oh, I didn't know. Yes, that is the instructions in the PR request.

Comment (sansyrox) [reply Cargo.toml]: oh, we should fix it then. :D

Also, rustfmt is failing one last time. We can merge it after that is addressed 😄

File: integration_tests/base_routes.py [hunk 2]
```diff
@@ -254,26 +255,26 @@ async def async_json_const_get():
 
 
 @app.get("/sync/param/:id")
-def sync_param(request: Request):
-    id = request.params["id"]
+def sync_param(Request):
```
Comment (sansyrox) [inline integration_tests/base_routes.py:258]: @carlosm27 , this should be `request: Request`

**Review by sansyrox — COMMENTED** [review]
Thanks for the great work @carlosm27 😄 
I have one comment

**Review by sansyrox — DISMISSED** [review]
@carlosm27 , thanks for the updated PR but there is still one merge conflict. We will be able to hopefully merge it after that 😅

**Review by sansyrox — APPROVED** [review]
Great work @carlosm27 🔥 

LGMT!

**Comment by carlosm27** [comment]
> Great work @carlosm27 🔥
> 
> LGTM!

Thank you @sansyrox for guiding me and helping me through the process.

**Pull request merged**